File: ABC229/b.py
a, b = input().split()

flg = False
for i in range(min(len(a), len(b))):
    if int(a[-1*(i+1)]) + int(b[-1*(i+1)]) >= 10:
        flg = True

if flg:
    print('Hard')
else:
    print('Easy')

File: ABC229/b.cpp
#include <stdio.h>
#include <iostream>
#include <vector>
#include <algorithm>
#include <math.h>

typedef long long ll;

using namespace std;

// ll pow(ll a, ll b)

int main()
{
  ll a, b;
  cin >> a >> b;

  bool flg = true;

  for (ll i = 18; i >= 0; i--)
  {
    ll wari = pow(10, i);
    ll a_wari = a / wari;
    ll b_wari = b / wari;

    if (a_wari + b_wari >= 10)
    {
      flg = false;
      break;
    }

    a = a % wari;
    b = b % wari;
  }

  cout << (flg ? "Easy" : "Hard") << endl;

  return 0;
}
